demo-app/src/data_generation.rs

```diff
@@ -88,3 +105,18 @@ impl QueryGenerator {
         Runtime::<MockContext>::encode_value_setter_query(query_message)
     }
 }
+
+impl Transaction<MockContext> {
+    pub fn new(msg: Vec<u8>, pub_key: MockPublicKey) -> Self {
+        let _hash = <MockContext as Spec>::Hasher::hash(&msg);
```

Comment:
(Question): Why do we create and drop the message hash here?

Reply:
This was a leftover, removed.